readme.md

```diff
@@ -10,5 +10,5 @@ ___
 
 **Result**
 
-[UPDATE DEMO LINK](https://<your_account>.github.io/<repo_name>/) <br>
-[UPDATE TEST REPORT LINK](https://<your_account>.github.io/<repo_name>/report/html_report/)
+[UPDATE DEMO LINK](https://<khrystynastyslo>.github.io/<layout_hello-world>/) <br>
+[UPDATE TEST REPORT LINK](https://<khrystynastyslo>.github.io/<layout_hello-world>/report/html_report/)
```

Comment:
<your_account>: `<>` - means variable name inside, you should replace `<>` too
Example https://khrystynastyslo.github.io/layout_hello-world/report/html_report/

Also, links do not work, test, please, before make PR